src/main/webapp/app/view/my-projects/my-projects.component.ts

```diff
@@ -130,7 +130,7 @@ export class MyProjectsComponent implements OnInit, OnDestroy {
       this.filteredProjects[this.filteredProjects.length - 1] &&
       this.filteredProjects[this.filteredProjects.length - 1].length % 3 !== 0
     ) {
-      this.filteredProjects[this.filteredProjects.length - 1].push({});
+      this.filteredProjects[this.filteredProjects.length - 1].push({} as Project);
```

Comment:
Warum benötigt es hier das `as Project` ?

Reply:
weil das `interface Project` mandatory Felder hat (z.b. `name`) und deshalb eigentlich nicht "leer" sein darf (`= {}`). hier aber nur als Platzhalter, damit wir in der Card-Ansicht immer schön auf 3 auffüllen können.